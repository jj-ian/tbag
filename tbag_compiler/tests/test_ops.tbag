true {
  intPrintLine(1 + 2);
  intPrintLine(1 - 2);
  intPrintLine(1 * 2);
  intPrintLine(100 / 2);
  intPrintLine(99);
  boolPrintLine(1 == 2);
  boolPrintLine(1 == 1);
  intPrintLine(99);
  boolPrintLine(1 != 2);
  boolPrintLine(1 != 1);
  intPrintLine(99);
  boolPrintLine(1 < 2);
  boolPrintLine(2 < 1);
  intPrintLine(99);
  boolPrintLine(1 <= 2);
  boolPrintLine(1 <= 1);
  boolPrintLine(2 <= 1);
  intPrintLine(99);
  boolPrintLine(1 > 2);
  boolPrintLine(2 > 1);
  intPrintLine(99);
  boolPrintLine(1 >= 2);
  boolPrintLine(1 >= 1);
  boolPrintLine(2 >= 1);
  endgame;
}

