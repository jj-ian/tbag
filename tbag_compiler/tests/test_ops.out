3
-1
2
50
99
false
true
99
true
false
99
true
false
99
true
true
false
99
false
true
99
false
true
true
